hedera-node/hedera-mono-service/src/main/java/com/hedera/services/contracts/execution/EvmTxProcessor.java

```diff
@@ -305,6 +242,12 @@ protected TransactionProcessingResult execute(
         }
     }
 
+    private boolean isSideCarTypeEnabled(final SidecarType sidecarType) {
```

Comment:
_Nit_ - maybe we want `EvmProperties` to include an `isSideCarTypeEnabled()` method?

Reply:
`Mirror-node` executions won't include generating sidecar files, so I think this property won't be visible and needed inside the evm library.